packages/webapp/src/components/ui/FormGenerator/FormFieldManager.ts

```diff
@@ -71,14 +87,14 @@ export class FormFieldManager {
 		for (const field of this.fields) {
 			if (isRequired(field) && !this.isFieldValueRecorded(field)) {
 				log(`validation error: field ${field.name} is required and not present`, this.value)
-				this.addFieldError(field.id, t('formGenerator.validation.required'))
+				this.addFieldError(field.id, t('formGenerator.validation.requiredField'))
 			}
 
 			if (field.type === ServiceFieldType.Number) {
 				const value = this.getRecordedFieldValue(field) as string
-				if (Number.isNaN(tryParseNumber(value))) {
+				if (isRequired(field) && Number.isNaN(tryParseNumber(value))) {
```

Comment:
does this mean it will only validate required numeric fields?

Reply:
The number field now uses a Mask component 'MaskedTextField' which doesn't allow anything besides numbers.
So the validation only happens if required and will show the error msg if the field is empty.